composer.json

```diff
@@ -26,7 +26,6 @@
         "php": ">=7.2",
         "cakephp/cakephp": "^4.0",
         "cakephp/chronos": "^2.0",
-        "cakephp/plugin-installer": "^1.0",
         "composer/composer": "^1.3",
```

Comment:
I wonder if the composer dependency was added just for plugin-installer.

Reply:
No, several classes are using composer classes

    use Composer\Json\JsonFile;

etc

Reply:
Makes sense!